paddle/scripts/docker/build.sh

```diff
@@ -43,5 +43,7 @@ cp -rv /woboq/data $WOBOQ_OUT/../data
     -o $WOBOQ_OUT \
     -p paddle:/paddle
 /woboq/indexgenerator/codebrowser_indexgenerator $WOBOQ_OUT
-
+cd /woboq
+make clean
```

Comment:
I want to build a docker env to test this commit, this may take some time, will this block you?